lib/generators/my_zipcode_gem/models_generator.rb

```diff
@@ -6,6 +6,7 @@ class ModelsGenerator < Base
     
     def initialize(*args, &block)
       super
+      migration_template 'migration.rb', "db/migrate/create_my_zipcode_gem_models.rb"
```

Comment:
Line is too long. [85/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.